index.html

```diff
@@ -203,6 +203,41 @@ <h1 class="introduction">I’m Mariane Glad to see you!</h1>
       <li class="topic item">Skills</li>
     </ul>
   </div>
+
+  <footer class="footer">
+    <div class="contact-me">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it
+      </p>
+    </div>
+
+    <div class="contact-forms">
+      <form class="message-form" action="https://formspree.io/f/form_id" method="post">
+        <ul>
+          <li class="input-li">
+            <input type="text" name="text" id="text-input" placeholder="Yeremias" maxlength="30" required>
+          </li>
+          <li class="input-li">
+            <input type="email" name="email" id="mail" placeholder="natajay14@gmail.com" required>
+          </li>
+          <li class="text-area-li">
+            <textarea name="msg" id="msg" maxlength="500" required>Write your messages here</textarea>
+
+          </li>
+          <li class="submit-li">
+            <button type="submit">Get it Touch</button>
+          </li>
+        </ul>
```

Comment:
- [ ]  Your form inputs are designed well 👏 . While I don't know the reason why you have put them in the list tag, I would like to tell you that you can still achieve **the same result without putting them in the List**. As developers, we have to keep our **codebase simple and clean** which makes it easy to understand 😃. Please remove your inputs from list tags.